java/ql/lib/semmle/code/java/dataflow/ExternalFlow.qll

```diff
@@ -317,7 +317,12 @@ private predicate summaryModelCsv(string row) {
       "java.io;File;true;getAbsolutePath;;;Argument[-1];ReturnValue;taint;manual",
       "java.io;File;true;getCanonicalPath;;;Argument[-1];ReturnValue;taint;manual",
       "java.nio;ByteBuffer;false;array;();;Argument[-1];ReturnValue;taint;manual",
+      "java.nio.file;Path;true;normalize;;;Argument[-1];ReturnValue;taint;manual",
+      "java.nio.file;Path;true;resolve;;;Argument[-1..0];ReturnValue;taint;manual",
       "java.nio.file;Path;false;toFile;;;Argument[-1];ReturnValue;taint;manual",
+      "java.nio.file;Path;true;toString;;;Argument[-1];ReturnValue;taint;manual",
+      "java.nio.file;Path;true;toUri;;;Argument[-1];ReturnValue;taint;manual",
+      "java.nio.file;Paths;true;get;;;Argument[0..1];ReturnValue;taint;manual",
```

Comment:
Since Java 11 there are the static methods [`Path.of(String, String...)`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/nio/file/Path.html#of(java.lang.String,java.lang.String...)) and [`Path.of(URI)`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/nio/file/Path.html#of(java.net.URI)) which as per `Paths` documentation should be preferred over the static `Paths` methods.
Should these methods therefore be covered here as well?

Reply:
We should add those, yes. I'll do it in a subsequent PR. Thanks for pointing this out!